src/objectTranslations/Behavior_Involvement__c-ca.objectTranslation

```diff
@@ -9,55 +9,55 @@
         <value><!-- Behavior Involvement --></value>
     </caseValues>
     <fields>
-        <help><!-- The Case that represents the event being reported. --></help>
-        <label><!-- Case --></label>
+        <help>El cas que representa l’esdeveniment denunciat.</help>
```

Comment:
@lifewithryan Just out of curiousity, is the ordering here how they are delivering it to us now? If so, that makes such a WORLD of difference!

Reply:
I'd have to double check the original branch. I think they might be doing that now but i still had to do an upload and retrieve to fix other items.

Reply:
and the answer is no, this was a result of deploying and retrieving.